modules/update-preparing-list.adoc

```diff
@@ -3,23 +3,25 @@
 // * updating/preparing_for_updates/updating-cluster-prepare.adoc
 
 [id="update-preparing-list_{context}"]
-= Removed Kubernetes APIs
+= Removed APIs
 
-{product-title} 4.19 uses Kubernetes 1.32, which removed the following deprecated APIs. You must migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-32[Kubernetes documentation].
+{product-title} 4.20 removed the following Kubernetes APIs. You must migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/[Kubernetes documentation].
 
-.APIs removed from Kubernetes 1.32
+.Kubernetes APIs removed from {product-title} 4.20
 [cols="2,2,2,1",options="header",]
 |===
 |Resource |Removed API |Migrate to |Notable changes
 
-|`FlowSchema`
-|`flowcontrol.apiserver.k8s.io/v1beta3`
-|`flowcontrol.apiserver.k8s.io/v1`
-|No
+|`MutatingWebhookConfiguration`
+|`admissionregistration.k8s.io/v1beta1`
+|`admissionregistration.k8s.io/v1`
+|link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#webhook-resources-v122[Yes]
 
-|`PriorityLevelConfiguration`
-|`flowcontrol.apiserver.k8s.io/v1beta3`
-|`flowcontrol.apiserver.k8s.io/v1`
-|link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#flowcontrol-resources-v132[Yes]
+|`ValidatingWebhookConfiguration`
+|`admissionregistration.k8s.io/v1beta1`
+|`admissionregistration.k8s.io/v1`
+|link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#webhook-resources-v122[Yes]
+
+// TODO: Check on ValidatingAdmissionPolicy / ValidatingAdmissionPolicyBinding from KCS - those aren't listed in the Kube deprecation guide for 1.22
```

Comment:
KCS also lists ValidatingAdmissionPolicy or ValidatingAdmissionPolicyBinding.

But the Kubernetes migration docs (and our OCP 4.9 docs) don't list ValidatingAdmissionPolicy or ValidatingAdmissionPolicyBinding for `admissionregistration.k8s.io/v1beta`. Only `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration`. Just checking whether it's appropriate to add these two or not?